tomviz/DataSource.cxx

```diff
@@ -42,18 +42,22 @@
 
 #include <cmath>
 #include <cstring>
+#include <limits>
 #include <sstream>
 
 namespace {
-void createOrResizeTiltAnglesArray(vtkDataObject* data)
+void createOrResizeTiltAnglesArray(
+  vtkDataObject* data,
+  vtkIdType maxSize = std::numeric_limits<vtkIdType>::max())
 {
   auto fd = data->GetFieldData();
   if (!fd->HasArray("tilt_angles")) {
     int* extent = vtkImageData::SafeDownCast(data)->GetExtent();
     int numTiltAngles = extent[5] - extent[4] + 1;
```

Comment:
I am guessing one of these numbers is coming out off, any realistic data set should have 1 - maybe a few thousand projections.